### DISCUSSION TITLE
1.2.0 NFT plots are showing up and earning points but not sending them to pool

### DISCUSSION THREAD

**Discussion by ZielReaver** [Chia-Network/chia-blockchain#7235] [Support]
I am using space pool and the devs over there already said that the GUI has a issue were it is not sending the points to the pool site. one of my three computers is sending but the other two just keep earning points but never sending in so the point balance never goes up. they all use same keys and launcher ID and pool contract.

please help as due to this it completely stops me from being able to pool

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
